teams.sql

```diff
@@ -0,0 +1,269 @@
+CREATE DATABASE nfl;
+
+USE nfl;
+
+CREATE TABLE teams (
+	id INT auto_increment,
+    location VARCHAR(255),
+    mascot VARCHAR(255),
+    abbreviation CHAR(3),
+    conference ENUM('AFC', 'NFC'),
+    division ENUM('north', 'south', 'east', 'west'),
+    PRIMARY KEY(id)
+);
+
+INSERT INTO teams(id, location, mascot, abbreviation, conference, division) VALUES (
+1,
```

Comment:
id is an auto_increment so you wouldn't provide a value yourself, you'd like the database handle that for you